boule/_sphere.py

```diff
@@ -145,28 +145,43 @@ def _check_geocentric_grav_const(self, geocentric_grav_const, value):
     @property
     def semiminor_axis(self):
         """
-        The semiminor axis of the sphere is equal to its radius. Added for
-        compatibility with pymap3d.
+        The semiminor axis of the sphere is equal to its radius.
         Definition: :math:`b = R`.
         Units: :math:`m`.
         """
         return self.radius
 
+    @property
+    def semimedium_axis(self):
+        """
+        The semimedium axis of the sphere is equal to its radius.
+        Definition: :math:`a = R`.
```

Comment:
```suggestion
```

Reply:
Yes, this is a little confusing. For sphere, we could use `a`, `b` and `c` like this
```
    @property
    def semiminor_axis(self):
        """
        The semiminor axis of the sphere is equal to its radius.
        Definition: :math:`c = R`.
        Units: :math:`m`.
        """
        return self.radius

    @property
    def semimedium_axis(self):
        """
        The semimedium axis of the sphere is equal to its radius.
        Definition: :math:`b = R`.
        Units: :math:`m`.
        """
        return self.radius

    @property
    def semimajor_axis(self):
        """
        The semimajor axis of the sphere is equal to its radius.
        Definition: :math:`a = R`.
        Units: :math:`m`.
        """
        return self.radius
```
This would use the same notation as `TriaxialEllipsoid`. However, since sphere doesn't use any of `a`, `b`, or `c`, I am ok with just removing the definition.

For `Ellipsoid`, it is more complicated, as `semiminor_axis` is defined as `b`, which is inconsistent with the notation for `TriaxialEllipsoid`. Either we can rename the variables to use `a`, `b` and `c`, or do as you suggest and just remove the definition.

My personal preference would be to use `a`, `b` and `c` consistently in all classes, but this is only because in my work (with things like the moment of inertia and rotation axis), this is standard notation, and all bodies are triaxial. I understand that `a` and `b` are used in a lot of textbooks or the Earth's biaxial shape, so I can understand why we might want to keep it that way too.

I suggest that we do whatever you decide. The `TriaxialEllipsoid` class is too developed at this point, and most people are just going to use the `Ellipsoid` class anyways.

Reply:
I agree with your point. It does sounds weird that for Ellipsoids b is the semiminor axis and for Triaxial b is the semimedium. But I think changing the symbols we use for them is kind of out of the scope of this PR. For this PR I think removing the definitions is enough.

I'm ok to keep the definitions of `a`, `b` and `c` in the Sphere, since they don't have a particular role.

So, what if we:
- Remove the mathematical definition of the semimedium in the Ellipsoid.
- Use `a`, `b` and `c` in the Sphere for the semimajor, semimedian and semiminor axis, respectively.